packages/charts/react-charting/src/components/DeclarativeChart/PlotlySchemaAdapter.ts

```diff
@@ -429,17 +429,17 @@ export const transformPlotlyJsonToSankeyProps = (
     }),
   };
 
-  const width: number = layout?.width || 440;
-  const height: number = layout?.height || 220;
+  const width: number = typeof layout?.width === 'number' ? layout?.width : 440;
```

Comment:
define a function _numberType(input, default) and similarly for string.

Reply:
okay, what should I set default values for places where it is not defined?

Reply:
I will take this change. Approving for now.